SwiftUI_DZ/SwiftUI_DZ/MessagesView/MessagesView.swift

```diff
@@ -0,0 +1,21 @@
+// MessagesView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Экран сообщений
+struct MessagesView: View {
+    // MARK: - Public Properties
+
+    var body: some View {
+        Image("1")
```

Comment:
```suggestion
        Image("")
```
Текстовки в константы